### DISCUSSION TITLE
Subclass from HTMLResponse on _TemplateResponse

### DISCUSSION THREAD

**Discussion by sponsfreixes** [encode/starlette#2272] [Ideas] [answered]
Hi! While reviewing a [PR to improve the support of FastAPI on jinja2-fragments](https://github.com/sponsfreixes/jinja2-fragments/pull/16), and trying to understand better how Starlette works with Jinja, I noticed that `startlette.templating._TemplateResponse` is subclassing from `starlette.responses.Response` and then setting the class attribute `media_type = "text/html"` (see [here](https://github.com/encode/starlette/blob/7e6c936487bc6bc57f15679fcadb8b9589d43636/starlette/templating.py#L26)).

Would you be open to a PR removing the media type, and replacing the parent class with `starlette.responses.HTMLResponse`?

This would make the type declaration more straightforward on our side (everything would just be instances or subclasses of `HTMLResponse`).

**Comment by Kludex** [comment, marked as answer]
> Would you be open to a PR removing the media type, and replacing the parent class with `starlette.responses.HTMLResponse`?

PR welcome.
